stingray/crossspectrum.py

```diff
@@ -629,6 +769,81 @@ def plot(self, labels=None, axis=None, title=None, marker='-', save=False,
         else:
             plt.show(block=False)
 
+    def classical_significances(self, threshold=1, trial_correction=False):
```

Comment:
This is good, I think we need to make a uniform treatment of trial correction across all our periodograms, including pulsar stuff. This is for a future PR.